src/NHibernate/Async/Loader/Loader.cs

```diff
@@ -334,7 +334,7 @@ private async Task<IList> DoQueryAsync(ISessionImplementor session, QueryParamet
 
 		internal async Task InitializeEntitiesAndCollectionsAsync(
 			IList hydratedObjects, DbDataReader reader, ISessionImplementor session, bool readOnly,
-			CacheBatcher cacheBatcher = null, CancellationToken cancellationToken = default(CancellationToken))
+			CacheBatcher cacheBatcher = null, CancellationToken cancellationToken)
```

Comment:
@maca88, some invalid code is generated here by the latest version of the AsyncGenerator, 0.13.0, by added a non-optional parameter after an optional one. Was the fix in 0.8.2.9 not done in latest versions of AsyncGenerator? (maca88/AsyncGenerator#115)

Reply:
Is was done but not released. Now I've released it in version 0.13.1.

Reply:
Thanks, upgrade chnaged to 0.13.1 and commit forced-pushed.